api/controllers/v0.3/AddressValidationAPI.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * A class that uses Service Objects to validate addresses.
+ * TODO: Finish the implementation.
+ */
+const AddressValidationApi = () => {
+  this.ALTERNATE_ADDRESSES_TYPE = 'alternate-addresses';
```

Comment:
What are alternate addresses--potential canonicalized versions of the address the user put in, or multiple residences, or...?

Reply:
Let's say you put in your address as "6th Avenue", the ILS can return an alternate address of "Avenue of the Americas"